main.go

```diff
@@ -84,8 +110,9 @@ func findReferences(ctx context.Context, rpc *jsonrpc2.Conn, rootDir, query stri
 		}
 		for _, result := range res {
 			if strings.Contains(result.URI, rootDir) {
-				fmt.Printf("\nFound references to type - %v:%v\n", result.URI, result.Range.Start.Line)
+				return result, true
```

Comment:
we'd probably actually want to accumulate these, and report something like: "usage of deprecated API at the following locations"

Reply:
Yeah face palm